app/src/controllers/groupOperations/utils.js

```diff
@@ -18,7 +18,7 @@ import { EXECUTE_GROUP_OPERATION } from './constants';
 
 export const validateItems = (items = [], validator, state) =>
   items.reduce((acc, item) => {
-    const error = validator(item, items, state);
+    const error = validator && validator(item, items, state);
```

Comment:
Should we put items without external issues to unlink modal? We can omit them  in prepareDataToSend